kobo/tasks.py

```diff
@@ -9,3 +10,4 @@ def perform_maintenance():
     Run daily maintenance tasks
```

Comment:
We talked before about adding an explicit task specific timeout - let's go do that now as we're adding more to it. It would be good to make sure it works as expected too. If you make it 1 second, will it really time out as expected? We don't want them retried either.

Reply:
I have added this with a 23 hour timeout [here](https://github.com/kobotoolbox/kpi/pull/4901/files#diff-06dc42ce86b33dc5058a5293a872d437206226e3f09f6fbbe32217ed459136a4). Note that I added a soft time limit, as well. This was necessary because if I only set the hard time limit it gets overridden by the global timeout.